lib/service/Service.ts

```diff
@@ -33,18 +37,20 @@ class Service {
    * Get general information about walli-server and the nodes it is connected to
    */
   public getInfo = async (): Promise<WalliInfo> => {
-    const { btcdClient, lndClient, ltcdClient } = this.serviceComponents;
+    const { btcdClient, lndClient, xudClient, ltcdClient } = this.serviceComponents;
     const version = packageJson.version;
 
     const btcdInfo = await btcdClient.getInfo();
     const ltcdInfo = await ltcdClient.getInfo();
     const lndInfo = await lndClient.getLndInfo();
+    const xudInfo = await xudClient.getXudInfo();
```

Comment:
You could start waiting for all promises to resolve with `await Promise.all()`

Reply:
I was thinking of doing that, but it's just ugly to work with arrays like that. Changed it tho to `await Promise.all()` as suggested.